### DISCUSSION TITLE
在 Feedly / Inoreader 中如何方便的使用「✂ 简悦的一键收藏」

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1559] [使用技巧]
> 下面的技巧来自社区的用户，细节请看这里 https://github.com/Kenshin/simpread/issues/1421

***

简悦 2.1.0 提供了「RSS 阅读器辅助」功能，可以在 RSS 阅读器中注入两个功能按钮，用于「全文输出」和「一键收藏」。但是目前版本的简悦只适配了 Inoreader 的列表模式。也就是说，除了列表模式，在分栏等其他模式下无法看到那两个功能按钮。

## Inoreader 的设置

不想把 Inoreader 改为列表模式的同学，目前有个办法可以在任意位置使用「一键收藏」：
1. 配置 Inoreader 的分享功能
![屏幕截图 2020-11-25 142958  2](https://user-images.githubusercontent.com/15528208/100191788-b7484c80-2f2b-11eb-9576-09687313674a.png)

2. 新增自定义站点，并保存
    - 名字：SimpRead
    - URL：http://localhost:7026/bookmark?mode=quick&url=[URL]&title=[TITLE]&port=7026
![屏幕截图 2020-11-25 143117](https://user-images.githubusercontent.com/15528208/100192030-26be3c00-2f2c-11eb-8b61-e6da58b3d87c.png)

3. 任意文章，直接分享
![屏幕截图 2020-11-25 143200  2](https://user-images.githubusercontent.com/15528208/100192207-77359980-2f2c-11eb-866d-75fbe4d4dff5.png)

打开简悦的「稍后读」，就可以在「未读列表」中看到刚才分享的文章了。

## 关联

- [如何进入 Feedly / Inoreader 才能看到简悦的注入 icon](https://github.com/Kenshin/simpread/discussions/1507)

**Comment by Kenshin** [comment]
### Feedly 的设置

http://localhost:7026/bookmark?mode=quick&url=${url}&title=${title}&port=7026

![image](https://user-images.githubusercontent.com/4997466/101018912-21559700-35a7-11eb-92a4-36b15af5c9d0.png)

**Reply by heliang12369** [reply]
哦，需要Feedly的收费版呀。。。。好吧，当我没看到过

**Comment by jy324** [comment]
Inoreader 分享文章以后在稍后读里打开，文章保存到本地吗？还是要打开原网页查看呢？

**Reply by Kenshin** [reply]
保存的只是 URL，还是需要在浏览器打开，然后才能生成快照的。

**Reply by jy324** [reply]
感觉有点鸡肋😂为什么不直接在浏览器打开呢？是先在稍后读中做个标记之后再剪藏吗

**Reply by Kenshin** [reply]
> 感觉有点鸡肋😂为什么不直接在浏览器打开呢？是先在稍后读中做个标记之后再剪藏吗

有的时候确实没有那么多的「时间」，这个做法相当于一个快捷收集功能。

我现在的做法：在浏览器加稍后读，然后空闲时间在 [Obsidian 中标注](https://github.com/Kenshin/simpread/discussions/4531#discussioncomment-3844881)
